todo_list_project/tasks/tests.py

```
@@ -186,20 +171,25 @@ def test_task_create_view(self):
+        and displays appropriate error messages in both English and Spanish.
         """
-        response = self.client.post(reverse('task_create'), {
-            'title': '',
-            'description': 'This task has no title'
-        })
-        self.assertEqual(response.status_code, 200)
-        self.assertFalse(Task.objects.filter(description='This task has no title').exists())
-        self.assertContains(response, 'This field is required.')
+        for lang_code in ['en', 'es']:
+            with self.subTest(lang=lang_code):
+                with override_settings(LANGUAGE_CODE=lang_code):
```

Ruff

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)